src/usyms.cpp

```diff
@@ -47,11 +87,79 @@ void Usyms::cache(const std::string &elf_file)
       pid_sym_[pid] = bcc_symcache_new(pid, &get_symbol_opts());
 }
 
-std::string Usyms::resolve(uint64_t addr,
-                           int32_t pid,
-                           const std::string &pid_exe,
-                           bool show_offset,
-                           bool show_module)
+#ifdef HAVE_BLAZESYM
+struct blaze_symbolizer *Usyms::create_symbolizer() const
```

Comment:
Can this be extracted out into a helper used by both this and ksyms?

Reply:
It could. Where do you suggest it would live, though?

Reply:
Could be in a new file or there is a symbols.h helper. Just an idea - this isn't a blocker.

Reply:
I just noticed that it actually can't be moved. The point of this method was that it accesses the `Usym` type's `config_` value, meaning that it needs to be defined within this type's purview.